.github/release-drafter.yml

```diff
@@ -0,0 +1,23 @@
+# Config for https://github.com/toolmantim/release-drafter
+name-template: 'Akka Management $NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: 'Cluster Bootstrap'
+    label: 't:bootstrap'
+  - title: 'Cluster HTTP Management'
+    label: 't:cluster-http'
+  - title: 'Discovery: AWS'
+    label: 't:aws'
+  - title: 'Discovery: Consul'
+    label: 't:consul'
+  - title: 'Discovery: Kubernetes'
+    label: 't:kubernetes'
+  - title: 'Discovery: Marathon'
+    label: 't:marathon'
+
+
+change-template: '- $TITLE [#$NUMBER](https://github.com/akka/alpakka/issues/$NUMBER) by [@$AUTHOR](https://github.com/$AUTHOR)'
```

Comment:
`$NUMBER` is a number of PR, not an issue 😞 
We use a rule, that subject of pull request  **MUST** start from the number of related issue.
Then the result of Release Drafter looks very informative.(example https://github.com/taymyr/lagom-openapi/releases/tag/v1.1.0)

Reply:
And `alpakka` looks like a copy&paste 😄

Reply:
PRs and issues shared the same sequence. You can refer to a PR that way.

Thanks for the copy&waste detection.